charmcraft/commands/init.py

```diff
@@ -106,17 +104,11 @@ def run(self, args):
         if not re.match(r"[a-z][a-z0-9-]*[a-z0-9]$", args.name):
             raise CommandError("{} is not a valid charm name".format(args.name))
 
-        if args.series is None:
-            series = "[kubernetes]  # TEMPLATE-TODO: change to an Ubuntu series if not using k8s"
-        else:
-            series = yaml.dump(args.series.split(","), default_flow_style=True)
```

Comment:
"series" should also be removed as the arguments of the command, right?

Reply:
Yep - fixed. Thanks.